src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    public LocalDate stringToLocalDate(String currentDateString) {
```

Comment:
Цей клас має лише робити звіт. Нормально що ми можемо винести певну логіку в окремі методи, але їх краще не робити доступними ззовні. Давай змінимо модифікатор доступу на `private` і підправимо назву
```suggestion
    private LocalDate parseDate(String currentDateString) {
```